src/tools/debugger/mhpdbgc.c

```diff
@@ -750,7 +784,9 @@ static void mhp_disasm(int argc, char * argv[])
 
    for (bytesdone = 0; bytesdone < nbytes; bytesdone += rc) {
        if (!(def_size & 4) && segmented) {
-          if (getsym_from_dos_segofs(seg,off+bytesdone))
+	  if (getsym_from_bios(seg, off + bytesdone))
+             mhp_printf ("%s:\n", getsym_from_bios(seg, off + bytesdone));
+	  else if (getsym_from_dos_segofs(seg, off + bytesdone))
              mhp_printf ("%s:\n", getsym_from_dos_segofs(seg,off));
```

Comment:
Is `+ bytesdone` missing here?

Reply:
Yes I wondered about that, it's existing code so if so it's already broken. Do you want me to add it?

Reply:
I've updated it now.